src/ResourceManager/Resources/Commands.Resources/ActiveDirectory/SetAzureADUserCommand.cs

```diff
@@ -0,0 +1,67 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.ActiveDirectory.Models;
+using Microsoft.Azure.Commands.Resources.Models.ActiveDirectory;
+using Microsoft.Azure.Graph.RBAC.Models;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.ActiveDirectory
+{
+    /// <summary>
+    /// Updates an existing AD user.
+    /// </summary>
+    [Cmdlet(VerbsCommon.Set, "AzureRmADUser"), OutputType(typeof(PSADUser))]
```

Comment:
ShouldProcess pattern, as with all cmdlets that make changes or have side effects